app/src/main/res/layout/activity_filtering_books.xml

```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".FilteringBooksActivity">
+
+<!--    Reset default filtering values-->
+    <Button
+        android:id="@+id/reset_button"
+        android:text= "@string/reset"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
```

Comment:
I think both buttons should be adjacent. Probably at the bottom so moving the reset, but it might get awkward if we do add a menu bar at the bottom, so I'm not sure.